jetty9-base/src/main/jetty-base/etc/java-util-logging.properties

```diff
@@ -0,0 +1,6 @@
+.level = INFO
```

Comment:
At a minimum, we need to make sure that this still works to overwrite the logging configuration:
```
env_variables:
  JAVA_USER_OPTS: -Djava.util.logging.config.file=/var/lib/jetty/webapps/root/WEB-INF/logging.properties
```
See discussion in issue #101

Also, maybe update the README.md as part of this PR specifying how to overwrite the default JUL config.